databricks-kube/src/context.rs

```diff
@@ -19,28 +19,59 @@ pub struct Context {
     api_secret_store: Arc<Store<Secret>>,
 }
 
-#[derive(Clone, Deserialize, Serialize, PartialEq, JsonSchema)]
+#[derive(Clone, Deserialize, Serialize, PartialEq, JsonSchema, Debug)]
 pub struct DatabricksAPISecret {
     pub databricks_url: String,
     pub access_token: String,
 }
 
+impl Default for DatabricksAPISecret {
+    fn default() -> Self {
+        Self {
+            databricks_url: String::from("default_url"),
+            access_token: String::from("default_token"),
+        }
+    }
+}
+
 #[derive(Clone, Deserialize, Serialize, PartialEq, JsonSchema)]
 pub struct OperatorConfiguration {
     pub api_secret_name: String,
+    pub default_poll_interval: String,
+    pub default_timeout_seconds: String,
+    pub default_requeue_interval: String,
+    pub secrets: Option<BTreeMap<String, String>>
+}
+
+impl Default for OperatorConfiguration {
+    fn default() -> Self {
+        Self {
+            api_secret_name: String::from("default_secret_name"),
+            default_poll_interval: String::from("250"),
+            default_timeout_seconds: String::from("10"),
+            default_requeue_interval: String::from("300"),
+            secrets: Some(BTreeMap::new())
+        }
+    }
 }
 
 impl Context {
-    pub fn get_databricks_url_token(&self) -> Option<(String, String)> {
-        let latest_secret = Self::latest_api_secret(self.api_secret_store.clone())?;
+    pub fn get_databricks_url_token(&self) -> Option<BTreeMap<String, String>> {
```

Comment:
Replace all of these individual field getters with:

```rust
pub fn get_operator_config(&self) -> OperatorConfiguration
pub fn get_api_secret(&self) -> DatabricksAPISecret
```

Handle all your serde in these methods such that you don't have to deal with things like marshalling the strings into ints at call site where you want to use the values.